runtime/common/src/lib.rs

```diff
@@ -189,6 +189,34 @@ impl pallet_staking::BenchmarkingConfig for StakingBenchmarkingConfig {
 	type MaxNominators = ConstU32<1000>;
 }
 
+/// Macro to set a value (e.g. when using the `parameter_types` macro) to either a production value
+/// or to an environment variable or testing value (in case the `fast-runtime` feature is selected).
```

Comment:
Maybe you should make more clear that the env variable needs to be set at compile time.